src/ullyses/calibrate_stis_data.py

```diff
@@ -807,7 +825,7 @@ def calibrate_stis_data(indir, yamlfile, dolog=True, logfile=None, outdir=None,
     for item in infiles:
         infile = os.path.join(indir, item)
         if not os.path.exists(infile):
-            raise FileNotFoundError(f"Input file defined in YAML, {infile_name}, cannot be found in direcotry {indir}")
+            raise FileNotFoundError(f"Input file defined in YAML, {infile}, cannot be found in direcotry {indir}")
```

Comment:
Typo here - direcotry

Reply:
Although this is already merged...

Reply:
I can add this in my misc. DR7 updates branch 👍

Reply:
Wait, sorry, I realized this was already fixed in later code. It will be a relic bug of the past then 😆